### PR TITLE
The End

### PR DESCRIPTION
**Pull request by Fox-McCloud**
This marks the end of an era. 

Completely removes the Process Scheduler, all of its defines, and all of its code, completely.

Also updates the Master Controller to the latest of what TG has......which is, well, 99.9% the same as what we already have; only real change is that some of the tick checking defines have changed and `stoplag` is now fully integrated into the MC so it can better control for lag.

Most references to "stoned MC" are now just "MC", as there's no need to differentiate the two anymore.

A big thank you to @Anasari, @Crazylemon64, @tigercat2000, @Markolie, @AffectedArc07 for assisting in transitioning, 100% over to this new system.

Local testing showed that the PS was actually using up a lot of CPU just to run, itself; my CPU at idle, prior to this change was 10-16 on the MC tab; after this change, it was only 3-6.

:cl: Fox McCloud
del: Removes the Process Scheduler
/:cl:

### CODE REVIEW

**Comment by AffectedArc07** [comment]
:clap:

**Comment by Kyep** [comment]
https://www.youtube.com/watch?v=JSUIQgEVDM4

**Comment by KasparoVy** [comment]
Insane

**Comment by MrMagolor** [comment]
RIP singh

You know goon probably uses the same process scheduler after all these years which is both funny and sad now that you think about it

File: code/__HELPERS/unsorted.dm [hunk 1]
```diff
@@ -1903,17 +1903,27 @@ var/mob/dview/dview_mob = new
...
+		return 1
+	if(!initial_delay)
+		initial_delay = world.tick_lag
+	. = 0
+	var/i = DS2TICKS(initial_delay)
+	do
+		. += CEILING(i*DELTA_CALC, 1)
+		sleep(i*world.tick_lag*DELTA_CALC)
+		i *= 2
+	while(TICK_USAGE > min(TICK_LIMIT_TO_RUN, Master.current_ticklimit))
```
Comment (ZeWaka) [inline code/__HELPERS/unsorted.dm]: This style of while-do statements are gross, in my opinion.

Comment (Kyep) [reply code/__HELPERS/unsorted.dm]: Still... I wish every game had a /proc/stoplag() you could just call.

**Pull request merged**